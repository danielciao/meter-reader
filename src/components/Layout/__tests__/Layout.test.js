import React from 'react';
import renderer from 'react-test-renderer';

import { Page, Section } from '../index';

describe('Layout', () => {
  it('should render Page correctly', () => {
    const result = renderer.create(<Page />).toJSON();

    expect(result).toMatchSnapshot();
  });

  it('should render Section correctly', () => {
    const result = renderer.create(<Section />).toJSON();

    expect(result).toMatchSnapshot();
  });
});
