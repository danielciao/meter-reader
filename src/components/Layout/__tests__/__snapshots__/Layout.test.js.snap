// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render Page correctly 1`] = `
<div
  className="sc-bdVaJa cdirPa"
/>
`;

exports[`Layout should render Section correctly 1`] = `
<section
  className="sc-bwzfXH bzRVtw"
/>
`;
